src/snapshot/parser.service.ts

```diff
@@ -632,6 +634,63 @@ export class ParserService {
     return buf;
   }
 
+  private getLifinityVaultsAndOwners() {
```

Comment:
so we treat lifinity as holder, right? (send them the rewards)

Reply:
yes, the lifinity should be considered as the holder, i.e., sending them rewards. they are able to sign the transactions with the treasury account as mentioned here